src/core/Akka.Remote/Transport/Transport.cs

```diff
@@ -21,6 +21,7 @@ public abstract class Transport
         public ActorSystem System { get; protected set; }
 
         public virtual string SchemeIdentifier { get; protected set; }
+        public virtual long MaximumPayloadBytes { get; protected set; }
```

Comment:
I just double checked to make sure that this is also exposed on the JVM transport, and it is: https://github.com/akka/akka/blob/master/akka-remote/src/main/scala/akka/remote/transport/Transport.scala#L92

So :+1: